script.js

```diff
@@ -0,0 +1,3 @@
+
+// VoiceRSS Javascript SDK
```

Comment:
what is this used for?

Reply:
this is a template from the author and I forget to delete the comment line.